recipes/b2/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "b2" %}
```

Comment:
This one's too generic of a name and all other package managers (except pip) use it for the b2 build system. Even their git repo is not named `b2` o avoid confusion.